Projects/Chat/Sources/Navigation/ChatNavigation.swift

```diff
@@ -71,7 +71,7 @@ public enum ChatNavigationViewName: TrackingViewNameProtocol {
 }
 
 public struct ChatNavigation<Content: View>: View {
-    @StateObject var router = Router()
+    @EnvironmentObject var router: Router
```

Comment:
I am not sure why we changed this from StateObject to EnvironmentObject.
Any reason to do so?

Reply:
Yeah I messed it up a bit :) fixed now!